src/frontend/src/components/TimeSlot.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box } from '@mui/system';
+import { getIcon } from '../utils/design-review.utils';
+
+interface TimeSlotProps {
+  text?: string;
+  fontSize?: string;
+  backgroundColor?: string;
+  icon?: string;
+  isModal: boolean;
```

Comment:
this should renamed to `small` (it's effectively making it smaller when it's in the modal from my understanding) because we don't wanna to tightly couple this component to how specifically we are using it rn